build/scripts/CrossBranchPorting.psm1

```diff
@@ -0,0 +1,147 @@
+<#
+    .SYNOPSIS
+    This script is used to backport a pull request to multiple branches.
+    .DESCRIPTION
+    This script is used to backport a pull request to multiple branches. It will create a new branch for each backport and create a pull request for each branch.
+    .PARAMETER PullRequestNumber
+    The number of the pull request to backport.
+    .PARAMETER TargetBranches
+    The list of branches to backport the pull request to.
+    .PARAMETER SkipConfirmation
+    Skip the confirmation prompt.
+#>
+function New-BCAppsBackport() {
+    param(
+        [Parameter(Mandatory=$true)]
+        [string] $PullRequestNumber,
+        [Parameter(Mandatory=$true)]
+        [string[]] $TargetBranches,
+        [Parameter(Mandatory=$false)]
+        [switch] $SkipConfirmation
+    )
+    Import-Module $PSScriptRoot/EnlistmentHelperFunctions.psm1
+
+    PrecheckBackport -TargetBranches $TargetBranches -PullRequestNumber $PullRequestNumber
+
+    # Get the pull request details
+    $pullRequestDetails = (gh pr view $PullRequestNumber --json title,number,body,headRefName,baseRefName,mergeCommit,potentialMergeCommit | ConvertFrom-Json)
+    Write-Host "Backport to: $($TargetBranches -join ",")" -ForegroundColor Cyan
+    Write-Host "Pull Request Source Branch: $($pullRequestDetails.headRefName)" -ForegroundColor Cyan
+    Write-Host "Pull Request Target Branch: $($pullRequestDetails.baseRefName)" -ForegroundColor Cyan
+    Write-Host "Pull Request Title: $($pullRequestDetails.title)" -ForegroundColor Cyan
+    Write-Host "Pull Request Description: `n$($pullRequestDetails.body)" -ForegroundColor Cyan
+
+    if (-not $SkipConfirmation) {
+        GetConfirmation -Message "Please review the about information and press (y)es to continue or any other key to stop"
+    }
+
+    # Get the list of existing pull requests
+    $existingOpenPullRequests = gh pr list --state open --json title,state,baseRefName,url | ConvertFrom-Json
+
+    # Get the current branch before starting to backport
+    $startingBranch = RunAndCheck git rev-parse --abbrev-ref HEAD
+
+    $pullRequests = @{}
+    $branchNameSuffix = "$PullRequestNumber/$(Get-Date -Format "yyyyMMddHHmmss")"
+    foreach($TargetBranch in $TargetBranches) {
+        try {
+            $title = "[$TargetBranch] $($pullRequestDetails.title)"
+            $body = "This pull request backports #$($pullRequestDetails.number) to $TargetBranch"
+            $body += "`r`n`r`nFixes [**Insert Work Item Number Here**]"
```

Comment:
Can you add `AB#` or at least `#` infront of `[**Insert Work Item Number Here**]`? :)